tracer/test/snapshots/Iast.StrictTransportSecurity.AspNetCore2.applicationxhtml%2Bxml.200.max-age%3D10%3Botherthingshttps.verified.txt

```diff
@@ -0,0 +1,54 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /iast/stricttransportsecurity,
+    Service: Samples.Security.AspNetCore2,
+    Type: web,
+    Tags: {
+      aspnet_core.route: iast/stricttransportsecurity,
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: iast/stricttransportsecurity,
+      http.status_code: 200,
+      http.url: http://localhost:00000/Iast/StrictTransportSecurity?contentType=application/xhtml%2Bxml&returnCode=200&hstsHeaderValue=max-age%3D10%3Botherthings&xForwardedProto=https,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.git.commit.sha: aaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbb,
+      _dd.git.repository_url: https://github.com/DataDog/dd-trace-dotnet,
+      _dd.iast.enabled: 1
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
```

Comment:
This one should have a vulnerability since the header is present but the value is not an integer?

Reply:
The decoded value would be max-age=10;otherthings which is valid because it has a value of 10. Whatever we find after the semicolon is not taken into account.